### ISSUE TITLE
Localhost Web UI

### ISSUE THREAD

**Issue by lescuer97** [comit-network/xmr-btc-swap#650]
Making a small Localhost Web UI to process the transactions directly wouldn't be a bad idea.  This way the user wouldn't have to copy paste at the end.

**Comment by thomaseizinger** [comment]
Are you talking about a Web UI for the user of the CLI?

**Comment by lescuer97** [comment]
Yes for the CLI, it could be simmilar to https://unstoppableswap.net/ but with the functionality of making the swap and seeing the state of the Swaps, keeping everything on the localhost so is a little more secure, or even give it access to the local network so you can check this on the phone. 

I think it would be a great addition for ease of use, maybe for a version 1.0.

**Comment by thomaseizinger** [comment]
GUIs are out of scope from our side.

The CLI can relatively easy be controlled from another process which makes it easy to build a GUI on top without having to modify the source code.

There is also the possibility of building a GUI in Rust and hooking directly into the library functions of the tool. That approach gives you more power and control but is also harder to implement and maintain.

I am going to close this as a wont-fix. If you are interested in building a GUI, I am happy to discuss these options more and provide some guidance.

**Comment by lescuer97** [comment]
Yes, I would totally like to implement a GUI for this and the guidance would be much appreciated.
